src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -206,16 +206,14 @@ def _check_config_and_set_final_flags(
 
         if strategy is not None:
             self._strategy_flag = strategy
-            if strategy == "ddp_cpu":
-                raise MisconfigurationException(
-                    "`Trainer(strategy='ddp_cpu')` is not a valid strategy,"
-                    " you can use `Trainer(strategy='ddp'|'ddp_spawn'|'ddp_fork', accelerator='cpu')` instead."
-                )
-            if strategy == "tpu_spawn":
-                raise MisconfigurationException(
-                    "`Trainer(strategy='tpu_spawn')` is not a valid strategy,"
-                    " you can use `Trainer(strategy='ddp_spawn', accelerator='tpu')` instead."
-                )
```

Comment:
Replacing these two by the general validation below. I don't think these specific errors are relevant anymore.